collect_app/src/androidTest/java/org/odk/collect/android/support/pages/ProjectSettingsDialogPage.kt

```diff
@@ -42,12 +46,12 @@ internal class ProjectSettingsDialogPage(rule: ActivityTestRule<*>) : Page<Proje
     }
 
     fun assertCurrentProject(name: String, icon: String): ProjectSettingsDialogPage {
-        onView(withId(R.id.current_project)).check(matches(withProject(name, icon)))
+        onView(allOf(withChild(withText(name)), withChild(withText(icon)), withContentDescription(getTranslatedString(R.string.using_project, name)))).check(matches(isDisplayed()))
```

Comment:
This of course isn't as exact as the previous `ProjectMatcher` as it checks that the view contains the two pieces of information we expect, but not that they are in the position we want them to be. I feel like the latter is better suited to a view test however. The next commit will deal with that.